.github/workflows/on-pr.yml

```diff
@@ -159,7 +159,7 @@ jobs:
       - name: Run Lint, Build, Test
         uses: mansagroup/nrwl-nx-action@v3
         env:
-          LOGGING_LEVEL: 'info'
+          LOG_LEVEL: 'info'
```

Comment:
@merrcury do we need to makes some updates on TF deployment aswell?

Reply:
No, We need to make this change just in secret Manager